sensor/common/detector/metrics/metrics.go

```diff
@@ -180,6 +180,27 @@ var (
 		Name:      "detector_network_flows_dropped_total",
 		Help:      "A counter of the total number of network flows that were dropped if the detector buffer was full",
 	})
+	detectorBlockScanCalls = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.SensorSubsystem.String(),
+		Name:      "block_scan_calls",
+		Help:      "A counter that tracks the operations in blocking scan calls",
+	}, []string{"Operation", "Path"})
+
+	scanCallDuration = prometheus.NewHistogram(prometheus.HistogramOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.SensorSubsystem.String(),
+		Name:      "scan_call_duration",
```

Comment:
Following https://prometheus.io/docs/practices/naming/, we always should have a unit appended to the name. Base unit is seconds, so here we have 2:0 for adding the unit.
```suggestion
		Name:      "scan_call_duration_milliseconds",
```

Reply:
Done